packages/vitest/src/runtime/rpc.ts

```diff
@@ -1,7 +1,9 @@
 import {
   getSafeTimers,
 } from '@vitest/utils'
+import type { BirpcReturn } from 'birpc'
 import { getWorkerState } from '../utils/global'
+import type { RuntimeRPC } from '#types'
```

Comment:
You can' access `#types` not in packages/ui

Reply:
@sheremet-va these changes are related to fixes for CI made by @userquin

Reply:
It doesn't mean that it's wrong

Reply:
@userquin i get you.